docs/specs/clients/core/identity/readme.md

```diff
@@ -0,0 +1,5 @@
+# Identity API
+
+## Description 
+
+Identity API exposes methods for registering and un-registering [Identity Keys](identity-keys.md) on the keychain, that will be used by other protocols (Namely [chat](link to chat) and [push](link to push)). Those keys represent an identity of a blockchain account on a specific device.
```

Comment:
We need to add the actual links instead of `link to chat` and `link to push` . Also it needs to be `notify` not `push`